### DISCUSSION TITLE
Beatmap selector list views

### DISCUSSION THREAD

**Discussion by Philosoph228** [ppy/osu#17922] [Ideas]
It would be amazing if there was a view mode switcher on beatmap selector screen. If beatmap was memorized well… by it's thumbnail it would be greater to find it by scrolling thumbnail grid like in direct downloader.

 ![Tux, the Linux mascot](https://i.imgur.com/eRXEPhk.png)

**Comment by peppy** [comment]
Can definitely agree with providing more flexibility at song select. Something to consider when refactoring things for the future design.
